logstash-core/src/main/java/co/elastic/logstash/api/Password.java

```diff
@@ -39,4 +39,8 @@ public String getPassword() {
     public String toString() {
         return "<password>";
     }
+
+    public String getValue() {
+        return getPassword();
```

Comment:
out of curiosity, is it necessary to have an alias method?

Reply:
Yes, because the original Ruby class Password has an `attr_reader` named `:value` which is used around the code